src/components/common/Layout/components/Head.tsx

```diff
@@ -0,0 +1,24 @@
+import NextHead from "next/head"
+
+interface Props {
+  ogTitle?: string
+  ogDescription?: string
+  imageUrl?: string
+  title?: string
```

Comment:
We don't need the title prop now, it was just used as a fallback if there's no ogTitle passed - now that it's a separate component anyway, we should make it required and remove the title prop